tests/console/aplay.pm

```diff
@@ -41,7 +41,11 @@ EOS
 
     start_audiocapture;
     assert_script_run("aplay ~/data/1d5d9dD.wav");
-    assert_recorded_sound('DTMF-159D');
```

Comment:
I just wonder if one can not catch an assert in test like this:

```perl
eval { assert_recorded_sound('DTMF-159D') };
record_soft_failure 'bsc#1048271' if ($@);
```

Reply:
Let me try that

Reply:
Nope, it didn't work: http://g226.suse.de/tests/2369#step/aplay/24

Reply:
something learned ;)

Reply:
the assert not only aborts the test, it also records a fail.